docs/user/tutorials/01-90-set-asynchronous-connection.md

```diff
@@ -1,27 +1,31 @@
-# Set Asynchronous Communication Between Functions
+# Send and Receive Cloud Events
```

Comment:
Maybe we can add a step after `4. Expose Function by creating the APIRule CR:` on how to obtain Kyma domain?
Like the 1st one [here](https://kyma-project.io/#/serverless-manager/user/tutorials/01-20-expose-function?id=procedure) in the kubectl tab.
Also can you change the apiVersion from `v2` to `v2alpha1`?

Reply:
done